### ISSUE TITLE
ci: jupyter lite deploy should use repo build, not from pypi

### ISSUE THREAD

**Issue by maartenbreddels** [widgetti/ipyreact#29]
If we do a release, the CI script installs from pypi the old version, we want it to be always up to date (I think)

**Comment by kolibril13** [comment]
> we want it to be always up to date 

do you mean the latest release version, or the latest version from the master branch?
I think we should always use the latest release version.

I just opened the latest JupyterLite, which is now running in 0.3.0 as expected.
But I guess that only happened because you trigger redeploy?
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/44469195/236759488-dbc72106-3982-43af-b702-6f14b0c6577d.png">

**Comment by maartenbreddels** [comment]
indeed, i re-triggered it. But if we re-deploy jupyter lite on every commit, then it's not in sync with the released version.

**Comment by kolibril13** [comment]
One quick and dirty solution: We could delay the pages build for 15 minutes after the commit, that should be enough time to have the latest pypi version, here is an action for that https://github.com/jakejarvis/wait-action

I can also investigate how other projects solve this problem and find a better solution in a few weeks.

**Comment by maartenbreddels** [comment]
Yeah, it's not urgent, but we'll forget it for the next release :)
